community/perl-ima-dbi/APKBUILD

```diff
@@ -4,39 +4,33 @@
 pkgname=perl-ima-dbi
 _pkgreal=Ima-DBI
 pkgver=0.35
-pkgrel=2
-pkgdesc="unknown"
-url="http://search.cpan.org/dist/Ima-DBI/"
+pkgrel=3
+pkgdesc="Database connection caching and organization"
+url="https://metacpan.org/pod/Ima::DBI"
 arch="noarch"
-license="GPL PerlArtistic"
-cpandepends="perl-dbi perl-class-data-inheritable perl-dbix-contextualfetch"
-cpanmakedepends="perl-test-pod perl-test-pod-coverage"
-depends="$cpandepends"
-makedepends="perl-dev $cpanmakedepends"
+license="GPL-1.0-or-later OR Artistic-1.0-Perl"
+depends="perl-dbi perl-class-data-inheritable perl-dbix-contextualfetch"
+makedepends="perl-dev perl-test-pod perl-test-pod-coverage"
 subpackages="$pkgname-doc"
-source="http://search.cpan.org/CPAN/authors/id/P/PE/PERRIN/$_pkgreal-$pkgver.tar.gz"
+source="https://cpan.metacpan.org/authors/id/P/PE/PERRIN/$_pkgreal-$pkgver.tar.gz"
 
 builddir="$srcdir/$_pkgreal-$pkgver"
 
 prepare() {
 	default_prepare
 
-	cd "$builddir"
 	PERL_MM_USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor
```

Comment:
Is `prepare()` the optimal place for this?
Usually `./configure`-style steps are done in `build()`.
Normally I wouldn't mention this, since we're modernizing anyway...